Palace/AppInfrastructure/TPPAppDelegate.m

```diff
@@ -1,228 +0,0 @@
-
-
-#if defined(FEATURE_DRM_CONNECTOR)
-#import <ADEPT/ADEPT.h>
-#import "TPPAccountSignInViewController.h"
-#endif
-
-// TODO: Remove these imports and move handling the "open a book url" code to a more appropriate handler
-#import "TPPXML.h"
-#import "TPPOPDSEntry.h"
-#import "TPPBookDetailViewController.h"
-#import "NSURL+NYPLURLAdditions.h"
-
-#import "TPPAppDelegate.h"
-
-@interface TPPAppDelegate()
-
-@property (nonatomic) AudiobookLifecycleManager *audiobookLifecycleManager;
-@property (nonatomic) TPPReachability *reachabilityManager;
-@property (nonatomic) TPPUserNotifications *notificationsManager;
-@property (nonatomic, readwrite) BOOL isSigningIn;
-@end
-
-@implementation TPPAppDelegate
-
-const NSTimeInterval MinimumBackgroundFetchInterval = 60 * 60 * 24;
-
-#pragma mark UIApplicationDelegate
-
-- (BOOL)application:(UIApplication *)app
-didFinishLaunchingWithOptions:(__attribute__((unused)) NSDictionary *)launchOptions
-{
-  [TPPErrorLogger configureCrashAnalytics];
-
-  // Perform data migrations as early as possible before anything has a chance to access them
-  [TPPKeychainManager validateKeychain];
-  [TPPMigrationManager migrate];
-  
-  self.audiobookLifecycleManager = [[AudiobookLifecycleManager alloc] init];
-  [self.audiobookLifecycleManager didFinishLaunching];
-
-  [TransifexManager setup];
-  [app setMinimumBackgroundFetchInterval:MinimumBackgroundFetchInterval];
-
-  [NSNotificationCenter.defaultCenter addObserver:self
-                                         selector:@selector(signingIn:)
-                                             name:NSNotification.TPPIsSigningIn
-                                           object:nil];
-
-  [[NetworkQueue shared] addObserverForOfflineQueue];
-  self.reachabilityManager = [TPPReachability sharedReachability];
-  
-  // Enable new reachability notifications
-  [Reachability.shared startMonitoring];
-  
-  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
-  self.window.tintColor = [TPPConfiguration mainColor];
-  self.window.tintAdjustmentMode = UIViewTintAdjustmentModeNormal;
-  [self.window makeKeyAndVisible];
-  
-  [[UITabBar appearance] setTintColor: [TPPConfiguration iconColor]];
-  [[UITabBar appearance] setBackgroundColor:[TPPConfiguration backgroundColor]];
-  [[UITabBarItem appearance] setTitleTextAttributes:@{NSFontAttributeName: [UIFont palaceFontOfSize:12.0]} forState:UIControlStateNormal];
-
-  [[UINavigationBar appearance] setTintColor: [TPPConfiguration iconColor]];
-  [[UINavigationBar appearance] setStandardAppearance:[TPPConfiguration defaultAppearance]];
-  [[UINavigationBar appearance] setScrollEdgeAppearance:[TPPConfiguration defaultAppearance]];
-  [[UINavigationBar appearance] setCompactAppearance:[TPPConfiguration defaultAppearance]];
-  if (@available(iOS 15.0, *)) {
-    [[UINavigationBar appearance] setCompactScrollEdgeAppearance:[TPPConfiguration defaultAppearance]];
-  }
-  
-  [self setUpRootVC];
```

Comment:
There's no `setUpRootVC` in the new code because initially this code was designed to run for SimplyE only and moved to `TPPAppDelegate+SE.swift`. We don't have this build target now and don't need this separation.